src/App.js

```diff
@@ -1,34 +1,154 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import fire from "./services/firebase";
+import Login from"./Login"
```

Comment:
functionally it won't matter, but for coding style consistency there should be a space after `from`